### PR TITLE
[Gradio] Add Gradio interface to 231-instruct-pix2pix-image-editing

### PR DESCRIPTION
**Pull request by itrushkin**
Replaces Jupyter widgets with Gradio interface for InstructPix2Pix tutorial.

ticket: _CVS-108281_

### CODE REVIEW

**Comment by itrushkin** [comment]
<p><code>share=True</code> uses network tunneling to Gradio server to forward the web interface. All calculations are executed on the host machine.</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1145/discussion/#comment-1600374981'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by ryanloney** [comment]
It works for me, but when increasing the steps, if there is a decimal point it fails with `TypeError: 'float' object cannot be interpreted as an integer`

Here's an image I made using Arc GPU in Windows 11: 
image

**Review by igor-davidyuk — APPROVED** [review]
LGTM

**Review by sbalandi — APPROVED** [review]

**Review by as-suvorov — APPROVED** [review]

File: notebooks/231-instruct-pix2pix-image-editing/231-instruct-pix2pix-image-editing.ipynb
Comment (aleksandr-mokrov) [inline notebooks/231-instruct-pix2pix-image-editing/231-instruct-pix2pix-image-editing.ipynb:1]: The output should be cleaned up. Especially installation output.

**Review by aleksandr-mokrov — CHANGES_REQUESTED** [review]

File: notebooks/231-instruct-pix2pix-image-editing/231-instruct-pix2pix-image-editing.ipynb
Comment (itrushkin) [reply notebooks/231-instruct-pix2pix-image-editing/231-instruct-pix2pix-image-editing.ipynb:1]: We usually have outputs for all cells to demonstrate the execution result in the documentation. For installation, I have added `-q` key for `pip install`.

**Review by aleksandr-mokrov — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Pull request merged**